.github/workflows/docker-build-push.yml

```diff
@@ -41,3 +41,13 @@ jobs:
         run: |
           docker build --build-arg SVC="${{ matrix.service_name }}" -t $GHCR/"${{ matrix.service_name }}":${{ github.sha }} -f "${{ matrix.service_name }}"/Dockerfile .
           docker push $GHCR/"${{ matrix.service_name }}":${{ github.sha }}
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Start services using Docker Compose
+        run: |
+          chmod +x ./build-tools/docker/check-docker-compose-services.sh
```

Comment:
Git supports +x for files, so you can save this step by doing this:
https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git/40979016

Reply:
Didn't know that. Will give it a go.

Reply:
applied git update-index --chmod=+x .ci/docker/check-docker-compose-services.sh and removed 'chmod' line..